ci-operator/config/openshift/operator-framework-olm/openshift-operator-framework-olm-master.yaml

```diff
@@ -84,7 +84,10 @@ tests:
     test:
     - as: test
       cli: latest
-      commands: JUNIT_DIRECTORY=${ARTIFACT_DIR} make e2e/olm SKIP='\[FLAKE\]'
+      commands: |
+        export JUNIT_DIRECTORY=${ARTIFACT_DIR}
+        export SKIP='\[FLAKE\]|an InstallPlan step contains a deprecated resource version'
```

Comment:
There's a character limit which the SKIP variable exceeded if inline with the e2e command.